### PR TITLE
basic/virt: treat "pouch" as a container type (id: pouch)

### PR DESCRIPTION
**Pull request by taskset**

### CODE REVIEW

**Comment by keszybz** [comment]
Needs a rebase. Also, there is nothing in this PR that would detect this new container type, afaics. You need to add a detection mechanism.

**Comment by w-simon** [comment]
> Needs a rebase. Also, there is nothing in this PR that would detect this new container type, afaics. You need to add a detection mechanism.

Ok.
Please take another look, thanks.

File: man/systemd-detect-virt.xml [hunk 1]
```diff
@@ -172,6 +172,11 @@
             <entry><varname>proot</varname></entry>
             <entry><ulink url="https://proot-me.github.io/">proot</ulink> userspace chroot/bind mount emulation</entry>
           </row>
+
+          <row>
+            <entry><varname>pouch</varname></entry>
+            <entry><ulink url="https://github.com/alibaba/pouch">Alibaba's Efficient Enterprise-class Rich Container Engine</ulink></entry>
```
Comment (poettering) [inline man/systemd-detect-virt.xml]: that sounds like an ad. "efficient", "enterprise-class", "rich". None of the other entries here use such words. Just say "Pouch Container Engine" or so

**Review by poettering — CHANGES_REQUESTED** [review]

**Comment by keszybz** [comment]
There should be just one commit, no merge commits in the PR — that's what we mean by a rebase.

File: man/systemd-detect-virt.xml (hunk 1 above)
Comment (taskset) [reply man/systemd-detect-virt.xml]: Okay, thanks.

**Comment by w-simon** [comment]
> There should be just one commit, no merge commits in the PR — that's what we mean by a rebase.
Okay, we really appreciate your guidance.

**Comment by keszybz** [comment]
LGTM.

**Pull request merged**